core/kazoo_documents/src/kzd_voicemail_box.erl

```diff
@@ -124,6 +126,13 @@ skip_greeting(Box) ->
 skip_greeting(Box, Default) ->
     kz_json:is_true(?KEY_SKIP_GREETING, Box, Default).
 
+-spec skip_envelope(doc()) -> boolean().
+-spec skip_envelope(doc(), Default) -> boolean() | Default.
```

Comment:
Please do not group `spec` line together. Move this line above its own function counterpart line.

Reply:
I think `scripts/evil_specs_remover.escript -k` will do the updates needed